<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTACheckECAMDisplayWidget</class>
 <widget class="QWidget" name="GTACheckECAMDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="CheckSubOptCB"/>
     </item>
     <item>
      <widget class="QComboBox" name="CheckValueCB"/>
     </item>
     <item>
      <widget class="QTextEdit" name="CheckValueTE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
